### ISSUE TITLE
again: colPos=18181 after moving a translated content-element

### ISSUE THREAD

**Issue by EnzephaloN** [FluidTYPO3/flux#1125]
TYPO3: 6.2.21
FLUX: 7.2.3

Pages-modul, Columns-view of all languages:
If I move a translated content-element around (arrow up  or arrow down), the content-element disappears, because it's colPos is set to 18181 .

**Comment by EnzephaloN** [comment]
Hey folks
I investigated a bit. I think the issue comes from ContentService.php row 241-243
`
		if (0 < $row['tx_flux_parent']) {
			$row['colPos'] = self::COLPOS_FLUXCONTENT;
		}
`
After commenting out $row['colPos'] = self::COLPOS_FLUXCONTENT; moving ce's by arrow was fixed.

**Comment by dneuge** [comment]
We encountered the same issue. We have several (~60) content elements which have `tx_flux_parent > 0` but where `tx_flux_column` is empty. The Flux parent ID doesn't make any sense as it mostly refers to a translated FCE which does not have (and never had) any container columns. The parent appears to be only set wrongly on translated content elements, maybe something is broken on creation of translated copies (such as reusing variables in a loop and missing to clean up on next iteration or something trivial like that)?

On some occasions, when moving affected content elements, the issue solves itself by setting `tx_flux_column` and `tx_flux_parent` to `NULL` (not empty/0! not sure if that's wrong as well). However, on most occasions, colPos 18181 gets set and the CE disappears from backend until colPos is manually fixed via DB.

Our FCE which being listed as a wrong Flux parent element has `tx_flux_children = 40` which (despite being set when it shouldn't) is inconsistent with the actual number of CEs referring to it.

This is quite a heavy inconsistency issue. I hope the root cause is something trivial and will be easy to fix for CEs created/translated in the future. Unfortunately, we can only start investigating this further starting next week as we first need to go live first with the affected website. Not a great feeling. :(

**Comment by dneuge** [comment]
The issue should be resolved by the linked pull request. It would be nice if this could be tested on a few other installations before merging. :) Please comment on #1163 for any issues which may have been introduced or not yet resolved but I feel like our changes handle the error completely.

**Comment by dr2okevin** [comment]
What is the satus of this Bug?
I have the same Problem with TYPO3 7.6.16 and flux 7.4.0

**Comment by dneuge** [comment]
Unfortunately, we (unexpectedly) did not find time to follow the review comments for the PR I submitted, last year. This year, I've changed jobs and I'm currently no longer working with TYPO3, so it doesn't make much sense for me to get back to the PR at this point as I've got nothing to test it with. I'd much appreciate if someone else could take over - I'm not aware of any issues with the PR's changes until I left the company.
